lib/install/management/create.go

```diff
@@ -29,6 +29,10 @@ import (
 	"github.com/vmware/vic/pkg/vsphere/tasks"
 )
 
+const (
+	uploadRetryCount = 5
```

Comment:
If it does not fix the overlying minor issues we can move to `retry.do` which was where I went first :smile: